README.md

```diff
@@ -117,57 +105,38 @@ require 'hanami/model'
 
 class Person
   include Hanami::Entity
-  attributes :name, :age
 end
 ```
 
 When a class includes `Hanami::Entity` it receives the following interface:
 
-  * `#id`
-  * `#id=`
   * `#initialize(attributes = {})`
-
-`Hanami::Entity` also provides the `.attributes` for defining attribute accessors for the given names.
+  * `#to_h` / `#to_hash`
 
 If we expand the code above in **pure Ruby**, it would be:
 
 ```ruby
 class Person
-  attr_accessor :id, :name, :age
-
   def initialize(attributes = {})
-    @id, @name, @age = attributes.values_at(:id, :name, :age)
+    @attributes = attributes
+  end
+
+  def to_h
+    @attributes
   end
+  alias to_hash to_h
 end
 ```
 
 **Hanami::Model** ships `Hanami::Entity` for developers's convenience.
 
-**Hanami::Model** depends on a narrow and well-defined interface for an Entity - `#id`, `#id=`, `#initialize(attributes={})`.
+**Hanami::Model** depends on a narrow and well-defined interface for an Entity - `#initialize(attributes={})`, `#to_h`.
```

Comment:
Is the resulting hash expected to have a key called `id`? If so, we might want to mention it here.